nncf/common/tensor_statistics/aggregator.py

```diff
@@ -46,34 +75,28 @@ def collect_statistics(self, model: TModel, graph: NNCFGraph) -> None:
         """
         if not self.statistic_points:
             return
-
+        batch_size = self.dataset.get_batch_size() or 1
+        if batch_size > 1 and self.is_model_has_no_batch_axis(graph):
+            nncf_logger.warning(BATCH_SIZE_MODEL_WARNING)
```

Comment:
As far as I understand, the warning will be printed every time statistics are collected. In this case, a user can see several warnings when calling `nncf.quantize`. Probably this check should be moved to function which call once for example `nncf.quantize()` or `quantize_impl`.

Reply:
done